sdk/turing/ensembler.py

```diff
@@ -279,6 +280,35 @@ def update(
             with the model. This will be passed to turing.ensembler.PyFunc.initialize().
             Example: {"config" : "config/staging.yaml"}
         """
+
+        # First we need to check if there are active job / router using the ensembler
+        # This check is done in the SDK to prevent the creation of new mlflow run
+
+        # check any active router version
+        relatedRouterVer = turing.Router.list_router_versions_with_filter(
+            ensembler_id=self._id, status=[RouterStatus.PENDING], is_current=False
```

Comment:
if we want to update the ensembler, we dont need to check deployed router version since it was already build with the old ensembler, if the user want to update the ensembler used by the router version, they just need to redeploy it to apply the new ensembler changes. wdyt?

Reply:
Good point. I was initially worried about what might happen to a router using the current version of the ensembler, should that version be deleted. But you're right, it shouldn't ever be an issue :)